packages/web3-common/src/deferred_promise.ts

```diff
@@ -48,6 +48,7 @@ export class DeferredPromise<T> implements Promise<T> {
 	}
 
 	public async catch<TResult>(
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		onrejected?: (reason: any) => TResult | PromiseLike<TResult>,
```

Comment:
instead of any could use `unknown` ?

Reply:
It's been enforced by the inherited native Promise api.